From 2527cd10d43a8eb0319584601a1e24f447ffe221 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Wed, 27 May 2020 13:57:47 -0700
Subject: [PATCH] dgit-maint-debrebase(7): Drop bad advice about upstream/ tag
 case

The first argument to the new-upstream subcommand is a version number,
not a tag name, so this instruction could never have been correct.

The user should not need usually to pass both the upstream version
number and the upstream/ tag name, either, because git-debrebase
should find it for them.

Reported-by: David Bremner <bremner@debian.org>
Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-debrebase.7.pod | 2 --
 1 file changed, 2 deletions(-)

diff --git a/dgit-maint-debrebase.7.pod b/dgit-maint-debrebase.7.pod
index 373fb2f7..fdbf2e8d 100644
--- a/dgit-maint-debrebase.7.pod
+++ b/dgit-maint-debrebase.7.pod
@@ -394,8 +394,6 @@ or if you have a working watch file
 
 =back
 
-replacing I<1.2.3> with I<upstream/1.2.3> if you imported a tarball.
-
 This invocation of git-debrebase(1) involves a git rebase.  You may
 need to resolve conflicts if the Debian delta queue does not apply
 cleanly to the new upstream source.
-- 
2.26.2

